### DISCUSSION TITLE
Server Shop Sign

### DISCUSSION THREAD

**Discussion by SuttonComputerServices** [Zrips/CMI#894] [Feature Requests]
CMI seems to realize that i am trying to use the essentails shop signs, but i am unable to make them work,
is there a way to create server shop signs, not for a chest shop, just a simple sign its self, layed out like:
[buy]
  32
Bread
$750

**Comment by CloudeLecaw** [comment]
Cmi does not have to relaise anything. your Essentials shop sign Needs to be able to relaise your used economy. normaly best why for this is using vault if the Plugin supports it

For the Moment CMI has no Shopsystem so there is no reason for nedd to look to other signshop Plugins


Your Question was is there a easy way to relaise a buy sign like you did?
Answer is - yes it is
Simply creat an interactivcommand that take your amount from the Player and give them the amout of items. Attach it to a sign somehwere. Make sure that the commands for this interactiv command have a cost to be executed. This means Nothing happens if they dotn have the Money!

Then look at the Sign and design your sign with /cmi se command
(if the sign is not designt allready ;) )

This will no Option for PlayersSignShops. 
If you want PlayerSignShops you have to use a working SignShop Plugin.

There is allready a Thread with the Theme PlayerShops.. My you write this usggestion there :3
https://github.com/Zrips/CMI/issues/260
